rollbar-client/test/Rollbar/ClientSpec.hs

```diff
@@ -85,7 +85,7 @@ spec = do
            { requestParams = HM.fromList [("user", "John Doe")] }
 
   describe "defaultNotifier" $
-    it "matches the package name and version" $ do
+    xit "matches the package name and version" $ do
```

Comment:
I know you mentioned in the PR description to "temporarily disable" this test in particular, should we be committing this change then?